packages/inferno-test-utils/src/index.ts

```diff
@@ -177,6 +189,83 @@ export function findVNodeWithType(vNodeTree: VNode, type: string | Function): VN
 	return findOneOf(vNodeTree, type, 'VNode', scryVNodesWithType);
 }
 
+// Jest Snapshot Utilities
+// Jest formats it's snapshots prettily because it knows how to play with the React test renderer.
+// Symbols and algorithm have been reversed from the following file:
+// https://github.com/facebook/react/blob/v15.4.2/src/renderers/testing/ReactTestRenderer.js#L98
+export function getTagNameOfVNode(inst: any) {
+	return (inst && inst.dom && inst.dom.tagName.toLowerCase()) ||
+		(inst && inst._vNode && inst._vNode.dom && inst._vNode.dom.tagName.toLowerCase()) ||
+		undefined;
+}
+
+export function createSnapshotObject(object: object) {
+	Object.defineProperty(object, '$$typeof', {
+		value: Symbol.for('react.test.json')
```

Comment:
Nope. That's actually how the Jest snapshot renderer knows that this object is special and that it should serialise it differently than a plain object.